resources/views/platform/components/cards/game.blade.php

```diff
@@ -25,21 +14,19 @@
         @if ($achievementsCount > 0 || mb_strpos($rawTitle, '~Z~') !== false)
             <!-- Progression Status -->
             @if ($highestProgressionStatus && $highestProgressionAwardDate)
-                <div class="my-1 flex items-center gap-x-1">
-                    @if (!$isEvent)
-                        <div class="rounded-full w-2 h-2" style="background-color: {{ $highestProgressionColor }}"></div>
-                    @endif
-
-                    <span>{{ $highestProgressionStatus }} {{ $highestProgressionAwardDate->format('j F Y') }}</span>
-                </div>
+                <x-cards.game.highest-progression-status
+                    :highestProgressionStatus="$highestProgressionStatus"
+                    :highestProgressionAwardDate="$highestProgressionAwardDate"
+                    :isEvent="$isEvent"
+                />
             @else
                 <div class="mb-2"></div>
             @endif
         @endif
 
         @if ($achievementsCount > 0)
             @if (!$isEvent)
-                <div class="leading-4">
+                <div class="-mt-0.5 leading-[0.98rem]">
```

Comment:
I noticed there was more bottom padding than top padding due to the line height of the text in this div. Remediated by this style change.